library/ssl_tls13_server.c

```diff
@@ -2269,6 +2269,10 @@ static int ssl_client_hello_fetch( mbedtls_ssl_context* ssl,
 
 static void ssl_debug_print_client_hello_exts( mbedtls_ssl_context *ssl )
 {
+#if !defined(MBEDTLS_DEBUG_C)
```

Comment:
ISTM that the entire function could be guarded by `MBEDTLS_DEBUG_C` and defined as a dummy if `!MBEDTLS_DEBUG_C`.